Processes/index.md

```diff
@@ -8,8 +8,6 @@ Create a pull request (PR) to submit content to the Docs Portal. The DevRel team
 ### Steps of Submission Process
 
 1. Learn about our [templates process](../Templates/index.md)
```

Comment:
For consistency with the how-to-submit... doc, add a period (.) at the end of every action bullet.

Reply:
Done